Sources/SwiftGRPCNIO/GRPCStatus.swift

```diff
@@ -25,11 +25,11 @@ public struct GRPCStatus: Error, Equatable {
 }
 
 public protocol GRPCStatusTransformable: Error {
-  func asGRPCStatus() -> GRPCStatus
+  func asGRPCStatus() -> GRPCStatus?
```

Comment:
Why does this need to return `GRPCStatus?` now?

Reply:
Internally, I'm using this to wrap any `Error` in a `WrappedError` that includes the location where it was thrown. However, `WrappedError` might or might not be wrapping another `GRPCStatusTransformable` — in the latter case it would have to return `nil` in its `asGRPCStatus()` implementation.

Reply:
Thanks for the explanation!

The current naming of `GRPCStatusTransformable` is at odds with with `asGRPCStatus()` returning an optional;  i.e. something conforming to `GRPCStatusTransformable` doesn't necessarily have to be transformable to `GRPCStatus`.

Could you have `WrappedError` conform to `GRPCStatusTransformable` and handle the underlying error not conforming to `GRPCStatusTransformable` there?

Reply:
Done; I guess I can simply return the "internal server error" myself.